dev/com.ibm.ws.logging/resources/com/ibm/ws/logging/internal/resources/LoggingMessages.nlsprops

```diff
@@ -168,6 +168,10 @@ JSON_ACCESS_LOG_NO_HEADER_NAME_SPECIFIED=TRAS3012W: Header name is missing for o
 JSON_ACCESS_LOG_NO_HEADER_NAME_SPECIFIED.explanation=JSON logging uses the logFormat attribute from accessLogging to select which http_access.log message fields to include. A header name must be supplied when using the %i or %o fields in the accessLogging logFormat attribute.
 JSON_ACCESS_LOG_NO_HEADER_NAME_SPECIFIED.useraction=Provide a header name for the field in the accessLogging logFormat attribute.
 
+FFDC_FILE_DELETION=TRAS3014I: Deleted {0} FFDC file(s) based on the maxFfdcAge value configured.
```

Comment:
Don't use (s) and move configured -> "Deleted {0} FFDC files based on the configured maxFfdcAge value."

Reply:
The message has been fixed, thanks.